dll/win32/browseui/internettoolbar.cpp

```diff
@@ -720,16 +715,10 @@ HRESULT CInternetToolbar::LockUnlockToolbars(bool locked)
     CDockSite                               *dockSite;
```

Comment:
Should this lock-unlock state apply to all toolbars at once and the status saved so that all future opened explorer windows get the new state?

Reply:
The locked/unlocked status should be saved, this is a setting that applies to new browseui windows that are opened and matches the behavior of Windows Server 2003. However, I don't think we need to refresh all the open browseui windows. Even Windows Server 2003 doesn't do that.